packages/@okta/vuepress-site/docs/guides/sign-in-to-spa-embedded-widget/main/javascript/nutrition.md

```diff
@@ -1,22 +1,20 @@
-This guide explains how to sign in to your SPA with the Okta Sign-In Widget.
+This guide explains how to sign in to a single page application (SPA) with the embedded Okta Sign-In Widget using JavaScript. It also provides other sign-in use case options and implementation considerations.
```

Comment:
```suggestion
This guide explains how to sign in to a single-page application (SPA) with the embedded Okta Sign-In Widget using JavaScript. It also provides other sign-in use case options and implementation considerations.
```

Reply:
I've changed this substantially: 

This guide explains authentication fundamentals using JavaScript and the embedded Okta Sign-In Widget, and provides a simple one-page SPA application to demonstrate a sign-in use case.

Reply:
dumb question, probably, but - isn't it redundant to say one-page SPA when SPA stands for single-page application?

Reply:
Additionally - you don't need a comma in front of and - I remove these because if it's not a compound sentence or if the second phrase isn't complex and unwieldy, there's no need for the comma.